### DISCUSSION TITLE
Welcome to Optimus Discussions!

### DISCUSSION THREAD

**Discussion by argenisleon** [ironmussa/Optimus#1068] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by metaalpha** [comment]
Hi @argenisleon ,
I am trying to connect to S3 bucket

from optimus import Optimus
op = Optimus("dask")

conn = op.connect.s3(endpoint_url="s3://my-S3-bucket-name")

AttributeError                            Traceback (most recent call last)
[/var/folders/g2/j5mknt0n50v7b2tsrbkxw13h0000gn/T/ipykernel_23400/2856264198.py](https://file+.vscode-resource.vscode-cdn.net/var/folders/g2/j5mknt0n50v7b2tsrbkxw13h0000gn/T/ipykernel_23400/2856264198.py) in <cell line: 1>()
----> 1 conn = Op.connect.s3(endpoint_url="s3://my-S3-bucket-name")

AttributeError: 'function' object has no attribute 's3'

Version : '23.5.0-beta'
OS : MacOS Montery

Please advise
